src/store/domains/globalSearch.js

```diff
@@ -0,0 +1,83 @@
+
+import { makeAutoObservable } from 'mobx';
+
+
+export default class GlobalSearch {
+  uiStore;
+
+  globalSearchAPI;
+
+  allFieldsTerm = '';
+
+  loading = false;
+
+  results = {
+    graphics: [],
+    paintings: [],
+    archivals: [],
+  };
+
+  error = null;
+
+  constructor(uiStore, globalSearchAPI) {
+    makeAutoObservable(this);
+
+    this.uiStore = uiStore;
+    this.globalSearchAPI = globalSearchAPI;
```

Comment:
Zwischenspeicherung von von außen durchgegebene Abhängigkeiten in Instanzvariablen, um innerhalb des Objekts darauf zugreifen zu können.